File: java/exer2/kart/Kart.java
package java.exer2.kart;

public class Kart {
  private String name;
  private Motor motor;
  String cilindradas;
  float velocidadeMaxima;

  public Kart(String name, String cilindradas, float velocidadeMaxima) {
    this.name = name;
    this.motor = new Motor(cilindradas, velocidadeMaxima);
  }

  public void pular() {
    System.out.println("O kart pulou");
  }

  public void soltarTurbo() {
    System.out.println("O kart soltou o turbo");
  }

  public void fazerDrift() {
    System.out.println("O kart fez um drift");
  }

  public Motor mostraInfo() {
    System.out.println("O motor tem " + cilindradas + "cilindradas e a velocidade max e " + velocidadeMaxima + "Km/h.");
    return motor;
  }

}

File: python/kart/kart.py
class Kart:
    def __init__(self, name, cilindradas, velocidadeMaxima):
        self.name = name
        self.cilindradas = cilindradas
        self.velocidadeMaxima = velocidadeMaxima
        self.motor = Motor(cilindradas, velocidadeMaxima)

    def pular(self):
        print("O kart pulou")

    def soltarTurbo(self):
        print("O kart soltou o turbo")

    def fazerDrift(self):
        print("O kart fez um drift")

    def mostraInfo(self):
        print(
            f"O motor tem {self.cilindradas} cilindradas e a velocidade máxima é {self.velocidadeMaxima} Km/h."
        )
        return self.motor


class Motor:
    def __init__(self, cilindradas, velocidadeMaxima):
        self.cilindradas = cilindradas
        self.velocidadeMaxima = velocidadeMaxima
